### ISSUE TITLE
OpenTracing: Make context switches optionally traced

### ISSUE THREAD

**Issue by sjoerdtalsma** [talsma-ict/context-propagation#71]
The opentracing module serves two purposes:
1. Propagate spans accross threads
2. Trace context switches

With a client I discovered that the traces became incredibly complex if each context-switch introduces two spans (capturing and reactivating the snapshot).
Because the propagation of spans is massively more important than tracing the context switches, let's make that second part optional (activated by a static method call and/or an environment variable).

**Comment by sjoerdtalsma** [comment]
Looking at the implementation, this can already be easily disabled:
```java
    private static final String SYS_DISABLED = "opentracing.contexttimer.disabled";
    private static final String ENV_DISABLED = SYS_DISABLED.toUpperCase().replace('.', '_');
    final String disabled = System.getProperty(SYS_DISABLED, System.getenv(ENV_DISABLED));
```

So defining an environment variable ``OPENTRACING_CONTEXTTIMER_DISABLED=true` or a Java system property `opentracing.contexttimer.disabled=true` will already disable the context switches!

**Comment by sjoerdtalsma** [comment]
On second thought, let's reverse the logic and make this _disabled by default_ instead of the other way around.

**Comment by sjoerdtalsma** [comment]
Fixed in #72
